orienteer-core/src/main/java/org/orienteer/core/widget/Widget.java

```diff
@@ -1,25 +1,55 @@
 package org.orienteer.core.widget;
 
+import org.orienteer.core.module.OWidgetsModule;
+
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-import org.orienteer.core.module.OWidgetsModule;
-
-import com.google.inject.BindingAnnotation;
-
 /**
  * Annotation for {@link AbstractWidget}'s to provide additional information
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE})
 public @interface Widget {
+	/**
+	 * @return id of Orienteer widget
+	 */
 	public String id();
+
+	/**
+	 * @return parent OClass for widget
+	 */
 	public String oClass() default OWidgetsModule.OCLASS_WIDGET;
+
+	/**
+	 * scheme - current widget applies only for scheme show widget
+	 * class  - current widget applies only for class show widget
+	 * document - current widget apply only for document show widget
+	 * @return domain for Widget.
+	 */
 	public String domain();
+
+	/**
+	 * @return tab name for widget. If tab is not empty creates new tab with given name in current domain for widget.
+	 */
 	public String tab() default "";
+
 	public int order() default 0;
+
+	/**
+	 * @return if true widget automatically enabled
+	 */
 	public boolean autoEnable() default false;
+
+	/**
```

Comment:
Selector depends on particular domain. For example for 'document' it's class name